src/components/shared/Datatable.vue

```diff
@@ -119,9 +119,17 @@
             getSortParam: {
                 type: Function,
                 default(sortOrder) {
-                    return sortOrder.map(sort => (`${sort.sortField}, ${sort.direction}`))
+                    return sortOrder.map(sort => (`${sort.sortField},${sort.direction}`)).join('|')
                 },
             },
+
+            /**
+             * Array for setting the initial sort order on the datagrid
+             *
+             */
+            'sort-order': {
```

Comment:
Isn't this handled in the vuetableConfig prop already?

Reply:
Ahh your right it can be set in the config, it can also be a prop though.. but thats removed now :)

Reply:
the wrapper component basically forwards any props to vuetable, and will also explode the `vuetableConfig` prop into separate props. So yes it could be it's own prop, but then again so could every thing in the `vuetableConfig` prop, I think it is neater to generally reduce the amount of props where possible and I think grouping all the official vuetable props into a single prop helps this.